flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/utils/BatchTestBase.scala

```diff
@@ -479,6 +479,6 @@ object BatchTestBase {
     conf.getConfiguration.setString(TABLE_EXEC_RESOURCE_HASH_JOIN_MEMORY, "2mb")
     conf.getConfiguration.setString(TABLE_EXEC_RESOURCE_SORT_MEMORY, "1mb")
     conf.getConfiguration.setString(TABLE_EXEC_RESOURCE_EXTERNAL_BUFFER_MEMORY, "1mb")
-    conf.getConfiguration.setString(TABLE_EXEC_SHUFFLE_MODE, ShuffleMode.PIPELINED.toString)
+    conf.getConfiguration.setString(TABLE_EXEC_SHUFFLE_MODE, ShuffleMode.BATCH.toString)
```

Comment:
@KurtYoung @JingsongLi I'd like to set `ShuffleMode` to `BATCH` in this test base to avoid resource deadlocks if a job has multiple slot sharing groups (1bb47697453af9045e59250d66ed2d1197597955). It's also the default behavior of blink batch executor.
Do you think it's valid to make this change?

Reply:
The commits in this PR alone don't cause the deadlock, correct? If so, then this change should be moved to the PR which modifies the actual runtime logic.

Reply:
Yes, we can remove it.

The deadlock happens in `RankITCase.testRankValueFilterWithRange` and `DistinctAggregateITCaseBase.testTwoDistinctAggWithGroupByAndCountStar` due to 
a combination of multiple reasons.
1. blink batch executor disables allVerticesInSameSlotSharingGroupByDefault (in this PR)
2. batch jobs in the given cases has multiple pipelined regions, it is a bug(FLINK-13708)
3. multiple pipelined regions lead to multiple slot sharing groups if allVerticesInSameSlotSharingGroupByDefault is disabled, as in #10007

I synced with @KurtYoung offline and the conclusion is blink will not generate a test job with multiple regions in the near future. So the tests would be Ok if FLINK-13708 is fixed. And we can drop this change in this way.

Reply:
The commit is dropped.